src/components/BackButton/StyledBackButton.tsx

```diff
@@ -0,0 +1,20 @@
+import { Button } from '..';
+import styled from 'styled-components';
+import BackLight from '../../views/HowToPlay/assets/BackLight.svg';
```

Comment:
Move the SVG file so it's close to the component that uses it.

```suggestion
import BackLight from './assets/BackLight.svg'
```